### DISCUSSION TITLE
Wouldn't it be better if the footer could be sticky even for pinned columns?

### DISCUSSION THREAD

**Discussion by dnjsgur0629** [KevinVandy/material-react-table#250] [Ideas]
In the following example, it looks weired for me that the footer of the pinned column is not sticky.
![stickyFooter](https://user-images.githubusercontent.com/65391273/206984527-431bf98e-461e-4bc8-8de7-e104ac5ed95f.gif)

If the enableStickyFooter props set true, it would be better if the pinned column's footer would also be sticky. What do you think about this?

[![Edit stickyFooter](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/stickyfooter-tg0xrw?fontsize=14&hidenavigation=1&theme=dark)

**Comment by KevinVandy** [comment]
Yeah this is probably just a bug

**Reply by dnjsgur0629** [reply]
Okay. Then will I make it an issue?
